mne/realtime/stim_server_client.py

```diff
@@ -9,7 +9,10 @@
 
 import numpy as np
 
-from ..utils import logger, verbose, fill_doc
+from ..utils import logger, verbose, fill_doc, deprecated
+
+RT_MSG = ('The realtime module is being deprecated from `mne-python` '
+          'and moved to its own repo, `mne-realtime`.')
```

Comment:
own repo -> own package?

Add "Use $ pip install mne-realtime" or something like that so people know how to get it